pkgs/top-level/all-packages.nix

```diff
@@ -8701,7 +8701,9 @@ in
 
   grail = callPackage ../development/libraries/grail { };
 
-  gtk-doc = callPackage ../development/tools/documentation/gtk-doc { };
+  gtk-doc = callPackage ../development/tools/documentation/gtk-doc {
+    python = python3;
```

Comment:
Can we just switch to python3 inside the expression?

Reply:
Sure. I'll push a fix.